controllers/items.js

```diff
@@ -1,85 +1,58 @@
-/*jshint esversion: 6 */
+"use strict";
 let fs = require('fs'),
     qs = require('querystring'),
-    config = require('../config');
+    config = require('../config'),
+    db = require('../dbconnect'),
+    getDBRecs =db.getDBRecs,
+    setDBRecs = db.setDBRecs,
+    rmDBRecs = db.rmDBRecs;
 
 exports.getAction = function (request, response) {
-    fs.exists(config.database.path, function (isExists) {
-        response.writeHead(200, {'Content-Type': 'application/json'});
-
-        if (false === isExists) {
-            response.end(JSON.stringify([]));
-
-        } else {
-            fs.createReadStream(config.database.path).pipe(response);
+    let docs;
+    getDBRecs((error, data) => {
+        if (error) {
+            docs = JSON.stringify(error.name + " : " + error.message);
         }
+        else {
+            docs = JSON.stringify(data);
+        }
+        console.log("DB Response: " + docs);
+        response.writeHead(200, {'Content-Type': 'application/json'});
```

Comment:
Below you used 503 http status code, which is in an error response, and this is correct. Use same approach here